kernel/reduction.ml

```diff
@@ -633,13 +633,29 @@ and eqappr cv_pb l2r infos (lft1,st1) (lft2,st2) cuniv =
         if Float64.equal f1 f2 then convert_stacks l2r infos lft1 lft2 v1 v2 cuniv
         else raise NotConvertible
 
+    | FArray (_u1,t1,ty1), FArray (_u2,t2,ty2) ->
+      (* FIXME compare instances *)
```

Comment:
@SkySkimmer would you mind taking care of this one?

Reply:
done